### PR TITLE
Adjusted pricing functions/added "display block" to map for now

### PR DESCRIPTION
**Pull request by amytschupruk**
Hey, I realized we were calculating the prices incorrectly. We were getting back "travel distance * hourly rate", when it should be "amount of hours * hourly rate" (for example, if it was a really close by trip but you wanted to have the bike for 24 hours, it would only charge $0.05 or however much the travel distance TIMES hourly rate was). Also just changed the map to display block so I could see it for messing around with markers...

### CODE REVIEW

File: dev/scripts/main.js [hunk 1]
```diff
@@ -249,29 +249,29 @@ bikeApp.getCityBikesPricing = function(userTime, distanceDurationMinutes, roundT
...
+		//multiply user hours to hourly price
 		const userHourlyPrice = (userTime * hourlyPrice).toFixed(2);
+		console.log(userHourlyPrice, "should be 0.66")
 		bikeApp.displayResults(distanceDurationMinutes, roundTripTime, userHourlyPrice, hourlyPrice); 
 	});
 };	
 
 
-bikeApp.displayResults = function (stationDistance, roundTripTravelTime, userRoundTripPrice, userPrice) {
+bikeApp.displayResults = function (stationDistance, roundTripTravelTime, userFinalPrice, pricePerHour) {
```
Comment (angelalagao) [inline dev/scripts/main.js]: ok can you just remove the pricePerHour parameter here since it is not being used

File: dev/scripts/main.js [hunk 2]
```diff
@@ -249,29 +249,29 @@ bikeApp.getCityBikesPricing = function(userTime, distanceDurationMinutes, roundT
 	.then(function(res) {
 		//24 hour pricing divided by 24 to obtain hourly rate
 		const hourlyPrice = (res.data.plans[1].price) / 24;
+		//multiply user hours to hourly price
 		const userHourlyPrice = (userTime * hourlyPrice).toFixed(2);
+		console.log(userHourlyPrice, "should be 0.66")
 		bikeApp.displayResults(distanceDurationMinutes, roundTripTime, userHourlyPrice, hourlyPrice);
```
Comment (angelalagao) [inline dev/scripts/main.js]: remove hourlyPrice argument here since it is not being used

File: dev/scripts/main.js [hunk 3]
```diff
@@ -249,29 +249,29 @@ bikeApp.getCityBikesPricing = function(userTime, distanceDurationMinutes, roundT
 	.then(function(res) {
 		//24 hour pricing divided by 24 to obtain hourly rate
 		const hourlyPrice = (res.data.plans[1].price) / 24;
+		//multiply user hours to hourly price
 		const userHourlyPrice = (userTime * hourlyPrice).toFixed(2);
+		console.log(userHourlyPrice, "should be 0.66")
```
Comment (angelalagao) [inline dev/scripts/main.js]: can also remove console.log

**Comment by amytschupruk** [comment]
I think that should be good now?

**Pull request merged**